crates/sdk/src/api/method/merge.rs

```diff
@@ -66,13 +65,26 @@ macro_rules! into_future {
 				};
 
 				let router = client.inner.router.extract()?;
-				let cmd = Command::Merge {
-					upsert,
+
+				let what = resource?.into_value();
+				let query = match (upsert, content) {
+					(true, Some(data)) => {
+						format!("UPSERT {} MERGE {}", what.to_sql(), data.to_sql())
```

Comment:
This can cause cause issues if `what.to_sql()` doesn't properly escape record id's with table names that conflict with a statement like `return:foo` or `select:1234567`. Maybe use params?